.github/workflows/test-cloud-platforms.yml

```diff
@@ -0,0 +1,149 @@
+          then
+            echo "Secrets to use an ARO cluster were configured in the repo"
+            echo "::set-output name=aks::true"
+          else
+            echo "Secrets to use an ARO cluster were not configured in the repo"
+            echo "::set-output name=aks::false"
+          fi
+
+  build-kubectl-gadget:
+    name: Build kubectl-gadget
+    # level: 0
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [linux, darwin, windows]
+        arch: [amd64, arm64]
+        exclude:
+          - os: windows
+            arch: arm64
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: 'main'
+      - name: Setup go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.18.3
+          cache: true
+        id: go
+      - name: Install debian packages
+        uses: ./.github/actions/install-debian-packages
+      - name: Build kubectl-gadget-${{ matrix.os }}-${{ matrix.arch }}
+        run: |
+          git checkout
+
+          # Prevent releases with -dirty suffix due to forgotten entries in
+          # .gitignore.
+          changes="$(git status --porcelain)"
+          if [ -n "$changes" ] ; then
+            echo "$changes"
+            exit 1
+          fi
+
+          kubectl_gadget=kubectl-gadget-${{ matrix.os }}-${{ matrix.arch }}
+
+          CONTAINER_REPO=${{ env.REGISTRY }}/${{ env.CONTAINER_REPO }} \
+          IMAGE_TAG=latest \
+          make $kubectl_gadget
+
+          # Prepare assets for release and actions artifacts
+          platform=$(echo ${kubectl_gadget} | cut -d- -f3-4)
+          mkdir $platform
+          cp $kubectl_gadget $platform/kubectl-gadget
+          cp LICENSE $platform/
+          tar --sort=name --owner=root:0 --group=root:0 \
+            -czf ${kubectl_gadget}.tar.gz -C $platform \
+            kubectl-gadget LICENSE
+          rm -rf $platform
+      - name: Add kubectl-gadget-${{ matrix.os }}-${{ matrix.arch }}.tar.gz as artifact.
+        uses: actions/upload-artifact@master
+        with:
+          name: kubectl-gadget-${{ matrix.os }}-${{ matrix.arch }}-tar-gz
+          path: /home/runner/work/inspektor-gadget/inspektor-gadget/kubectl-gadget-${{ matrix.os }}-${{ matrix.arch }}.tar.gz
+
+  test-integration-aks:
+    name: Integration tests on AKS
+    # level: 1
+    needs: [check-secrets, build-kubectl-gadget]
+    if: needs.check-secrets.outputs.aks == 'true'
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os-sku: [Ubuntu]
+        arch: [amd64, arm64]
+    steps:
+    - uses: azure/login@v1
+      name: Login to Azure
+      with:
+        creds: ${{ secrets.AZURE_AKS_CREDS }}
+    - name: Create AKS cluster
+      shell: bash
+      run: |
+        cluster_name=${{ secrets.AZURE_AKS_CLUSTER_PREFIX }}${{ matrix.arch }}-${{ matrix.os-sku }}
+
+        node_size='Standard_D2s_v5'
+        if [ ${{ matrix.arch }} = 'arm64' ]; then
+          node_size='Standard_D2ps_v5'
+        fi
```

Comment:
Could you please roughly describe in the documentation why we need two diff node sizes?

Reply:
You are right. Maybe it is better to have a comment here.